plugins/modules/win_copy.py

```diff
@@ -90,6 +90,10 @@
 - Because win_copy runs over WinRM, it is not a very efficient transfer
   mechanism. If sending large files consider hosting them on a web service and
   using M(ansible.windows.win_get_url) instead.
+- When using become with a user that is not an Administrator, the remote temp
+  directory must be set to a one that is accessible to both the become and
+  connection user with C(ansible_remote_tmp). See the examples for more
+  more information.
```

Comment:
That extra note is great! Just a couple grammatical bits worth tidying up.
```suggestion
- When using become with a user that is not an Administrator, the remote temp
  directory must be set to one that is accessible to both the become and
  connection user with C(ansible_remote_tmp). See the examples for more
  information.
```